packages/cra-template-typescript/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "cra-template-typescript",
+  "version": "1.0.0",
+  "keywords": [
+    "react",
+    "create-react-app",
+    "template",
+    "typescript"
+  ],
+  "description": "The base TypeScript template for Create React App.",
+  "main": "template.json",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/facebook/create-react-app.git",
+    "directory": "packages/cra-template-typescript"
+  },
+  "license": "MIT",
+  "engines": {
+    "node": ">=8"
```

Comment:
Following up on https://github.com/facebook/create-react-app/pull/7839/files, should this be `"node": ">=8.10"` or something?

Reply:
Yes, will do.